bin/make-zip.sh

```diff
@@ -21,9 +21,9 @@ output 2 "Creating archive... 🎁"
 
 ZIP_FILE=$1
 
-build_files=$(ls dist/*/*.{js,css})
+build_files=$(find dist/ \( -name '*.js' -o -name '*.css' \))
```

Comment:
Curious about the changes here, was something not working as expected?

Reply:
Yes, the command was failing for me in Linux. Not all shells support the `/*/*.` glob so find should be much more portable.

Reply:
Thanks for fixing that up @superdav42